### ISSUE TITLE
why include ckeditor in this widget as folder assets?!

### ISSUE THREAD

**Issue by NabiKAZ** [2amigos/yii2-ckeditor-widget#64]
Why after 1.0.3 version, remove `ckeditor/ckeditor` from require composer and add ckeditor in assets folder? You don't thinks better ckeditor project separate from yii widget as it's in older version?

**Comment by tonydspaniard** [comment]
because ckeditor didn't provide the full source as a separated source. The only way to include full source + full plugins was to added within the project.

**Comment by NabiKAZ** [comment]
Full ckeditor source there are here:

https://github.com/ckeditor/ckeditor-releases
OR
https://packagist.org/packages/ckeditor/ckeditor

You can use assets publisher yii, to access and use this packages in vendor.
In this case, the users can have self update ckeditor core. and you can focus on yii widget.

**Comment by tonydspaniard** [comment]
i know, but as you can see on the ckeditor package, the full version is on dev branches and we couldn't install them at the time of creating the widget with minimum stability to 'stable'. So I decided to add a version to it. 

If you can solve the issue with a PR, that would also include all plugins available, I will be more than pleased to merge it.

**Comment by NabiKAZ** [comment]
I did not notice the problem.
I install ckeditor from composer:

`composer require ckeditor/ckeditor`
composer.json
```
"require": {
  "ckeditor/ckeditor": "^4.5"
},
```

composer.lock
```
        {
            "name": "ckeditor/ckeditor",
            "version": "4.5.9",
            "source": {
                "type": "git",
                "url": "https://github.com/ckeditor/ckeditor-releases.git",
                "reference": "e3d4dba95343f1b33d8d9a08b78cd080efa0cdc1"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/ckeditor/ckeditor-releases/zipball/e3d4dba95343f1b33d8d9a08b78cd080efa0cdc1",
                "reference": "e3d4dba95343f1b33d8d9a08b78cd080efa0cdc1",
                "shasum": ""
            },
:
:
```

And I see `vendor\ckeditor\ckeditor` about 7.10MB
And so 68 directories in `plugins` folder.
Seem this is full and stable [released](https://github.com/ckeditor/ckeditor-releases/releases) version.
What's the problem?!

**Comment by tonydspaniard** [comment]
In the past I wasn't able to add the full version of it. It was not possible. Do you mind making a PR and make sure the tests pass? I would be more than grateful, otherwise, I need to setup some time aside to do it my self.

**Comment by tonydspaniard** [comment]
fixed on version 2.0. Thanks.
